api/weather-data.js

```diff
@@ -34,3 +34,44 @@ function getWeatherData(callback) {
     }
   });
 }
+
+module.exports.getWeatherDataByCoords = getWeatherDataByCoords;
+
+function getWeatherDataByCoords(lat, long, callback) {
+  var url = 'http://api.met.no/weatherapi/locationforecast/1.9/?lat=' + lat + ';lon=' + long;
+  request(url, function(error, response, body) {
+    if (error) {
+      callback(error);
+    } else if (response.statusCode != 200) {
+      callback(new Error(response.statusCode));
+    } else {
+      var xmlDoc = libxmljs.parseXml(body);
+
+      var currentTimePeriod = xmlDoc.get('//product/time/location');
+
+      var temperature = currentTimePeriod.get('temperature/@value').value();
+      var windSpeed = currentTimePeriod.get('windSpeed/@mps').value();
+      var effectiveTemperature = 13.12 +
+                                 0.6215 * temperature -
+                                 11.37 * Math.pow(windSpeed, 0.16) +
+                                 0.3965 * temperature * Math.pow(windSpeed, 0.16);
+
+      // Symbols are weird in the met.no API
+      // The first time data-point doesn't seem to have symbol data,
+      // but the next one usually does.
+      var symbol = xmlDoc.get('//product/time/location/symbol/@number').value();
```

Comment:
Yes, how this works is not really documented by met.no.

All queries I have made to the API looks something like this:

```
<product class="pointData">
    <time datatype="forecast" from="2016-05-10T15:00:00Z" to="2016-05-10T15:00:00Z">
        <location altitude="485" latitude="60.1000" longitude="9.5800">
            <temperature id="TTT" unit="celsius" value="17.9"/>
            <windDirection id="dd" deg="166.5" name="S"/>
            <windSpeed id="ff" mps="1.3" beaufort="1" name="Flau vind"/>
            <windGust id="ff_gust" mps="2.7"/>
            <humidity value="55.3" unit="percent"/>
            <pressure id="pr" unit="hPa" value="1018.2"/>
            <cloudiness id="NN" percent="76.0"/>
            <fog id="FOG" percent="-0.0"/>
            <lowClouds id="LOW" percent="19.1"/>
            <mediumClouds id="MEDIUM" percent="0.9"/>
            <highClouds id="HIGH" percent="74.1"/>
            <dewpointTemperature id="TD" unit="celsius" value="8.8"/>
        </location>
    </time>
    <time datatype="forecast" from="2016-05-10T14:00:00Z" to="2016-05-10T15:00:00Z">
        <location altitude="485" latitude="60.1000" longitude="9.5800">
            <precipitation unit="mm" value="0.0" minvalue="0.0" maxvalue="0.0"/>
            <symbol id="PartlyCloud" number="3"/>
        </location>
    </time>
    ....
</product>
```

The first time-element seems to always have structure similar to the one above, while the next has symbol.
The timedelta of the first element always seems to be 0, while the next one always is a period that overlaps with the timestamp of the first element.
The first `xmlDoc.get()` will find the first `time`-element, the second will find the first `time`-element that has symbol data.